### PR TITLE
feat: v13 `Permissions`.

### PR DESCRIPTION
**Pull request by WiseDevHelper**
**Please describe the changes this PR makes and why it should be merged:**

[Demo](https://deploy-preview-705--discordjs-guide.netlify.app/popular-topics/permissions.html)

This PR updates the `Permissions` section of the guide for the `v13` release.

**Status**
- [x] Update `Permissions`.
- [x] Update `Permissions (extended)`.
- [x] Update code samples.

Please let me know if there is more to change. I'll be happy to work on them.

### CODE REVIEW

**Review by almostSouji — CHANGES_REQUESTED** [review]
Also needs to address code samples (remove the v11/v12 subfolders)

File: guide/popular-topics/permissions.md [hunk 1]
```diff
@@ -119,16 +123,34 @@ guild.channels.create('new-channel', {
 });
 ```
 
+### Editing overwrites
+
+To edit permission overwrites on the channel with a provided set of new overwrites, you can use the `.edit()` method. This method allows passing an array or Collection of <DocsLink path="class/PermissionOverwrites">PermissionOverwrites</DocsLink>.
+
+```js
+// editing overwrites with PermissionOverwriteOptions
```
Comment (Danktuary) [inline guide/popular-topics/permissions.md:131]:
```suggestion
```

File: guide/popular-topics/permissions.md [hunk 2]
```diff
@@ -167,7 +189,7 @@ channel.lockPermissions()
 
 ### Permissions after overwrites
 
-discord.js features two utility methods to easily determine the final permissions for a guild member or role in a specific channel: `.permissionsFor()` on the <DocsLink path="class/GuildChannel?scrollTo=permissionsFor">GuildChannel</DocsLink> class and `.permissionsIn()` on the <DocsLink path="GuildMember?scrollTo=permissionsIn">GuildMember</DocsLink> and <DocsLink path="class/Role?scrollTo=permissionsIn">Role</DocsLink> classes. Both return a <DocsLink path="class/Permissions">Permissions</DocsLink> object.
+Discord.js features two utility methods to easily determine the final permissions for a guild member or role in a specific channel: `.permissionsFor()` on the <DocsLink path="class/GuildChannel?scrollTo=permissionsFor">GuildChannel</DocsLink> class and `.permissionsIn()` on the <DocsLink path="GuildMember?scrollTo=permissionsIn">GuildMember</DocsLink> and <DocsLink path="class/Role?scrollTo=permissionsIn">Role</DocsLink> classes. Both return a <DocsLink path="class/Permissions">Permissions</DocsLink> object.
```
Comment (Danktuary) [inline guide/popular-topics/permissions.md:192]:
```suggestion
There are two utility methods to easily determine the final permissions for a guild member or role in a specific channel: `.permissionsFor()` on the <DocsLink path="class/GuildChannel?scrollTo=permissionsFor">GuildChannel</DocsLink> class and `.permissionsIn()` on the <DocsLink path="GuildMember?scrollTo=permissionsIn">GuildMember</DocsLink> and <DocsLink path="class/Role?scrollTo=permissionsIn">Role</DocsLink> classes. Both return a <DocsLink path="class/Permissions">Permissions</DocsLink> object.
```

**Review by Danktuary — COMMENTED** [review]
Small nitpicks, looks good otherwise

**Review by almostSouji — APPROVED** [review]

**Pull request merged**